openshift/templates/pipeline/eagle-api-pipeline.bc.json

```diff
@@ -0,0 +1,88 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "creationTimestamp": null,
+        "labels": {
+            "app": "${NAME}-pipeline",
```

Comment:
Change this one to ${GROUP_NAME} and add that variable to the others at the bottom of the file. Helps with teardown, especially when we have multiple deploys of things, as the app label is the main selector used for that.